addons/l10n_in/views/report_invoice.xml

```diff
@@ -30,17 +30,19 @@
             </td>
         </xpath>
 
-        <xpath expr="//h2" position="replace">
-            <h2>
-                <span t-if="o.move_type == 'out_invoice' and o.state == 'posted'" t-field="o.journal_id.name"/>
-                <span t-if="o.move_type == 'out_invoice' and o.state == 'draft'">Draft <span t-field="o.journal_id.name"/></span>
-                <span t-if="o.move_type == 'out_invoice' and o.state == 'cancel'">Cancelled <span t-field="o.journal_id.name"/></span>
-                <span t-if="o.move_type == 'out_refund'">Credit Note</span>
-                <span t-if="o.move_type == 'in_refund'">Vendor Credit Note</span>
-                <span t-if="o.move_type == 'in_invoice'">Vendor Bill</span>
-                <span t-field="o.name"/>
-            </h2>
-        </xpath>
+        <t t-if="o.company_id.account_fiscal_country_id.code == 'IN'">
+            <xpath expr="//h2" position="replace" >
```

Comment:
I don't understand how it can work, I would expect that all override are put together, and only then the view is rendered with the values. I guess it does or this PR would not work.

For this use case, I would for example do something like:

```xml
<xpath expr="//h2" position="replace" >
    <t t-if="o.company_id.account_fiscal_country_id.code == 'IN'">
        <h2>...new code</h2>
    </t>
    <t t-else="">$0</t>
</xpath>
```

Reply:
Probably better than mine indeed.